src/plugins/discover/server/ui_settings.ts

```diff
@@ -242,15 +243,27 @@ export const getUiSettings: () => Record<string, UiSettingsParams> = () => ({
     category: ['discover'],
     schema: schema.boolean(),
   },
+  [ROW_HEIGHT_OPTION]: {
+    name: i18n.translate('discover.advancedSettings.params.rowHeightTitle', {
+      defaultMessage: 'Maximum lines of document explorer cell',
+    }),
+    value: 3,
+    category: ['discover'],
+    description: i18n.translate('discover.advancedSettings.params.rowHeightText', {
+      defaultMessage:
+        'The maximum lines that a cell in a table should occupy. Auto height might be enabled by setting value to -1. Single option could be enabled by setting value to 0.',
```

Comment:
I'd suggest to change to
`Displayed lines of a document explorer's row. Auto height can be enabled by setting the value to -1. The explorer's single option can be enabled by setting value to 0.`

Reply:
In looking at other documents, the term "row height" seems to be appropriate here.  It's also the word you use in the pop-up.  Here is a suggestion:

**Row height in the Document Explorer**
The number of lines to allow in a row.  A value of -1 automatically adjusts the row height to fit the contents. A value of 0 displays the content in a single line.

I'm a little confused over the term Single and whether it is the same as setting Custom to 1.

Reply:
@gchaps sounds good, yes "single". Thing is you can configure the height in the explorer to show just 1 line in 2 ways. Either you could choose **Single** or **Custom** with a height of 1. They look a bit different in terms of truncation, and work differently under the hood